.github/ISSUE_TEMPLATE/issue report.yml

```diff
@@ -87,46 +146,42 @@ body:
     validations:
       required: true
   - type: markdown
+    id: general-information-logs
     attributes:
       value: |
-        When providing logs, please keep the following things in mind.
-        1. **DO NOT** use external paste services.
+        When providing logs, please keep the following things in mind:
+        1. **DO NOT** use external paste services. If logs are too large to paste into the field, upload them as text files.
         2. Please provide complete logs.
-          - For server logs, include everything you think is important plus *10 lines before and after*
+          - For server logs, ensure to capture all relevant information, encompassing both the events leading up to and following the occurrence of the issue. Typically, providing 10 *lines preceding and succeeding* the problem should be adequate.
```

Comment:
Should also tell users to include logs even if they think nothing is relevant

Reply:
That is written in the beginning as well: `Fill in as much of the template as possible. When you are unsure about the relevancy of a section, do include the information requested in that section. Only leave out information in sections when you are completely sure about it not being relevant.`

You think we need to repeat ourselves here?